extensions/wikia/Chat2/ChatAjax.class.php

```diff
@@ -45,7 +45,7 @@ static public function getUserInfo() {
 		}
 
 		$user = User::newFromId( $data['user_id'] );
-		if ( empty( $user ) || !$user->isLoggedIn() || $user->getName() != urldecode( $wgRequest->getVal( 'name', '' ) ) ) {
+		if ( empty( $user ) || !$user->isLoggedIn() || $user->getName() != $wgRequest->getVal( 'name', '' ) ) {
```

Comment:
For legibility, let's convert this conditional into a multiline one with one condition per line, as per [conventions](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#conditional-logic) :)